packages/core/test/ui_builder_tests.js

```diff
@@ -248,13 +255,15 @@ tap.test('groupPatterns - orders patterns when provided from md', function(
     subtypePatterns: {},
   });
 
+  // Should be sorted by order and secondly by name
   patternlab.patterns.push(
```

Comment:
Changed the ordering while creation to demonstrate that ordering by order + name works